extension/chrome/elements/compose.ts

```diff
@@ -41,9 +41,12 @@ export class ComposeView extends View {
   public readonly removeAfterClose: boolean;
   public readonly disableDraftSaving: boolean;
   public readonly debug: boolean;
+  public readonly useFullScreenSecureCompose: boolean;
   public readonly isReplyBox: boolean;
   public readonly replyMsgId: string;
   public readonly replyPubkeyMismatch: boolean;
+  public readonly composeMethod: 'reply' | 'forward' | undefined;
+  public readonly thunderbirdMsgId: number;
   public replyOption?: ReplyOption;
```

Comment:
there is already `replyOption` parameter which is similar to `composeMethod`, probably we can use it without adding `composeMethod` param?

Reply:
Thanks for the suggestion - I reviewed this case and I think I can just use `replyOption` for me not to add yet another url parameter to parse. Please kindly check if i'll do it the right way.

Reply:
Upon checking, the typings would not align to `replyOption` when the moment I will need to pass it to the `addTripleDotQuoteExpandFooterAndQuoteBtn`. the typings...`'reply'|'forward'`... of the thunderbird seems to align very well with `addTripleDotQuoteExpandFooterAndQuoteBtn` so even if I had removed the requirement of passing URL parameter in the compose.ts, the replyOption accepts `'a_reply'|'a_forward'` so there is that restriction that I see.